src/h5web/visualizations/heatmap/ColorMapSelector.tsx

```diff
@@ -27,29 +55,51 @@ function ColorMapSelector(): JSX.Element {
     shallow,
   ]);
 
-  const gradientStyles = {
-    option: (
-      selectStyles: CSSProperties,
-      { data: { label } }: { data: { label: ColorMap } }
+  const customStyles = {
+    option: (provided: CSSProperties) => ({
+      ...provided,
+      display: 'flex',
+      justifyContent: 'space-between',
+    }),
+    singleValue: (
+      provided: CSSProperties,
+      { data: { value } }: { data: { value: ColorMap } }
     ) => ({
-      ...selectStyles,
-      backgroundImage: generateCSSLinearGradient(INTERPOLATORS[label], 'right'),
-      marginTop: '1px',
-      marginBottom: '1px',
+      ...provided,
+      backgroundImage: generateCSSLinearGradient(INTERPOLATORS[value], 'right'),
+      padding: '0.25rem',
+      color: INTERPOLATORS[value](1),
```

Comment:
Setting the color of the text to the far-right color works well for most colormaps in background.

Reply:
Why not just use the same visual as for the options with the text on the left of the gradient? You may even be able to reuse your custom option component.